src/components/Navbar/navbar.module.scss

```diff
@@ -1,5 +1,8 @@
+$font_color: #354E57;
```

Comment:
You should add this color variables to a variables.scss and then import it when needed.
So you should create this `variables.scss`file inside `styles` folder, and then import it to this file like 
````
import "../../styles/variables.scss"
````
(note: path may not be correct)